src/collectors/ceph/ceph.py

```diff
@@ -12,22 +12,32 @@
 
 """
 
-try:
-    import json
-except ImportError:
-    import simplejson as json
-
+import json  # No need for simplejson fallback b/c ceph py modules are >=2.6
 import glob
 import os
 import subprocess
-
+import re
 import diamond.collector
+import diamond.convertor
+from diamond.collector import str_to_bool
+
+
+# Metric name/path separator
+_PATH_SEP = "."
```

Comment:
So my main concern is that this is private at all. Technically the way a metric is separated might mot be the role of the collector, but in fact the handler. It's just odd that a specific collector is choosing to define this as opposed to it being a global in the base class.